seismostats/analysis/b_significant.py

```diff
@@ -0,0 +1,399 @@
+"""This module contains functions for the evaluation if a one dimensional
+b-value series varies significantly, following Mirwald et al, SRL (2024)
+"""
+
+import numpy as np
+import warnings
+from scipy.stats import norm
+
+from seismostats.analysis.bvalue import ClassicBValueEstimator
+from seismostats.analysis.bvalue.base import BValueEstimator
+from seismostats.analysis.avalue import AValueEstimator
+from seismostats.utils._config import get_option
+
+
+def est_morans_i(values: np.ndarray,
+                 w: np.ndarray | None = None,
+                 mean_v: float | float | None = None) -> tuple:
+    """
+    Estimates the nearest neighbor auto correlation (Moran's I) of the values.
+
+    Args:
+        values:     Values for which the autocorrelation is estimated.
+        w:          Weight matrix, indicating which of the values are
```

Comment:
call this "spatial weight matrix" or something? we have weight matrices all over the repository which indicate something different...

Reply:
Good point- changed the docstrings and called it neighbor matrix. Still left the argument "w", as "n" seemed kind of wrong.